lib/modem_key_mgmt/modem_key_mgmt.c

```diff
@@ -15,6 +15,11 @@
 #define ENABLE 1
 #define DISABLE 0
 
+/* Even though it's defined in minimal libc, the declaration of strtok_r appears to be missing in
+ * some cases, so we forward declare it here.
+ */
+extern char *strtok_r(char *, const char *, char **);
```

Comment:
This needs a fix like `gcf_sms.c` too though.

Reply:
yup just noticed that. guess it makes sense to also forward declare in the other files?